<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Clientes</class>
 <widget class="QWidget" name="Clientes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>671</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1010</width>
    <height>671</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1010</width>
    <height>671</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1010</width>
     <height>671</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1010</width>
     <height>671</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1010</width>
     <height>671</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #ADCE74;</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page1_ListarClientes">
    <widget class="QPushButton" name="btnInicio_clientes">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>96</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">	background-color: #F09200;
	color: #004b23;
	border: 1.5px solid #F09200;
	border-style: outset;
	border-radius: 5px;</string>
     </property>
     <property name="text">
      <string> In√≠cio</string>
     </property>
     <property name="icon">
      <iconset resource="imagens.qrc">
       <normaloff>:/img/img/Home.png</normaloff>:/img/img/Home.png</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>711</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #004b23;</string>
     </property>
     <property name="text">
      <string>Lista de Clientes</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btnNovo_cliente">
     <property name="geometry">
      <rect>
       <x>884</x>
       <y>30</y>
       <width>96</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">	background-color: #F09200;
	color: #004b23;
	border: 1.5px solid #F09200;
	border-style: outset;
	border-radius: 5px;</string>
     </property>
     <property name="text">
      <string> Novo</string>
     </property>
     <property name="icon">
      <iconset resource="imagens.qrc">
       <normaloff>:/img/img/Add.png</normaloff>:/img/img/Add.png</iconset>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget_clientes">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>951</width>
       <height>371</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #E6F7E6;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>100</number>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>460</y>
       <width>951</width>
       <height>191</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <fontweight>DemiBold</fontweight>
      </font>
     </property>
     <property name="title">
      <string>Pesquisar por:</string>
     </property>
     <widget class="QLabel" name="lblTotalClientes">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>221</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <fontweight>ExtraBold</fontweight>
       </font>
      </property>
      <property name="text">
       <string>Clientes:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPesquisarCliente">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>50</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <fontweight>DemiBold</fontweight>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(196, 196, 196);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="imagens.qrc">
        <normaloff>:/img/img/pesquisar.png</normaloff>:/img/img/pesquisar.png</iconset>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtNifPesquisa">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#DAF7D9;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>NIF</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtClientePesquisa">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>301</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#DAF7D9;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Cliente</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLimparPesquisa">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>50</y>
        <width>51</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <fontweight>DemiBold</fontweight>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">
background-color: rgb(196, 196, 196);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="imagens.qrc">
        <normaloff>:/img/img/Refresh.png</normaloff>:/img/img/Refresh.png</iconset>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page2_RegistarClientes">
    <widget class="QTextEdit" name="txtObservacoes">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>550</y>
       <width>411</width>
       <height>65</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#DAF7D9;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNIF">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>280</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#DAF7D9;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>9</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtNome">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>411</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#DAF7D9;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>100</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Data do registo</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>260</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Telefone</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtRegisto">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(208, 208, 208);</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>530</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Observa√ß√µes</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>711</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #004b23;</string>
     </property>
     <property name="text">
      <string>Registo de Clientes</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Cliente</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtTelefone">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>280</y>
       <width>161</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#DAF7D9;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="maxLength">
      <number>9</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btnVoltar_clientes">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>96</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">	background-color: #F09200;
	color: #004b23;
	border: 1.5px solid #F09200;
	border-style: outset;
	border-radius: 5px;</string>
     </property>
     <property name="text">
      <string> Voltar</string>
     </property>
     <property name="icon">
      <iconset resource="imagens.qrc">
       <normaloff>:/img/img/Previous.png</normaloff>:/img/img/Previous.png</iconset>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtData">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>120</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(208, 208, 208);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Registo</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>260</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>NIF</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>410</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Endere√ßo</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtEmail">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>360</y>
       <width>411</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#DAF7D9;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>340</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCancelar">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>130</y>
       <width>96</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">	background-color: #F09200;
	color: #004b23;
	border: 1.5px solid #F09200;
	border-style: outset;
	border-radius: 5px;</string>
     </property>
     <property name="text">
      <string> Cancelar</string>
     </property>
     <property name="icon">
      <iconset resource="imagens.qrc">
       <normaloff>:/img/img/Delete.png</normaloff>:/img/img/Delete.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btnEliminar">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>250</y>
       <width>96</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">	background-color: #F09200;
	color: #004b23;
	border: 1.5px solid #F09200;
	border-style: outset;
	border-radius: 5px;</string>
     </property>
     <property name="text">
      <string> Eliminar</string>
     </property>
     <property name="icon">
      <iconset resource="imagens.qrc">
       <normaloff>:/img/img/Apagar.png</normaloff>:/img/img/Apagar.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btnModificar">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>190</y>
       <width>96</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">	background-color: #F09200;
	color: #004b23;
	border: 1.5px solid #F09200;
	border-style: outset;
	border-radius: 5px;</string>
     </property>
     <property name="text">
      <string> Alterar</string>
     </property>
     <property name="icon">
      <iconset resource="imagens.qrc">
       <normaloff>:/img/img/Modify.png</normaloff>:/img/img/Modify.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="btnGuardar">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>70</y>
       <width>96</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">	background-color: #F09200;
	color: #004b23;
	border: 1.5px solid #F09200;
	border-style: outset;
	border-radius: 5px;</string>
     </property>
     <property name="text">
      <string> Guardar</string>
     </property>
     <property name="icon">
      <iconset resource="imagens.qrc">
       <normaloff>:/img/img/Save.png</normaloff>:/img/img/Save.png</iconset>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtEndereco">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>430</y>
       <width>411</width>
       <height>65</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#DAF7D9;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblErro">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>30</y>
       <width>261</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtRegisto</tabstop>
  <tabstop>txtData</tabstop>
  <tabstop>txtNome</tabstop>
  <tabstop>txtNIF</tabstop>
  <tabstop>txtTelefone</tabstop>
  <tabstop>txtEmail</tabstop>
  <tabstop>txtEndereco</tabstop>
  <tabstop>txtObservacoes</tabstop>
  <tabstop>tableWidget_clientes</tabstop>
  <tabstop>btnInicio_clientes</tabstop>
  <tabstop>txtNifPesquisa</tabstop>
  <tabstop>btnVoltar_clientes</tabstop>
  <tabstop>btnNovo_cliente</tabstop>
  <tabstop>txtClientePesquisa</tabstop>
  <tabstop>btnCancelar</tabstop>
  <tabstop>btnEliminar</tabstop>
  <tabstop>btnModificar</tabstop>
  <tabstop>btnGuardar</tabstop>
  <tabstop>btnPesquisarCliente</tabstop>
 </tabstops>
 <resources>
  <include location="imagens.qrc"/>
 </resources>
 <connections/>
</ui>
